Add deposit, withdraw, and borrow to Matomo analytics

jahabeebs opened the pull request:
closes #591 

## Description
- Adds events for deposit, withdraw, and borrow to Matomo

I think I need access to the Matomo dashboard to actually test this

pi0neerpat reviewed (changes requested):
- Add user address
- Ensure transaction is submitted before sending tracking event

jahabeebs commented:
> * Add user address
> 
>     * Ensure transaction is submitted before sending tracking event

@pi0neerpat 

- Added a check for the tx hash success to be sure the transaction went through
- Added user address
- Added amounts borrowed/deposited/withdrawn (you mentioned we'd add this in the future but it was trivial to add this so if you prefer I can take it out or leave it in)
